addon/serializers/children.js

```diff
@@ -0,0 +1,12 @@
+import OsfSerializer from './osf-serializer';
+
+export default OsfSerializer.extend({
+    serialize: function(snapshot, options) {
```

Comment:
Like this?
https://github.com/emberjs/data/blob/v2.5.1/addon/serializers/json-api.js#L397